src/algorithm/SCAN-OMP.cpp

```diff
@@ -0,0 +1,99 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-21, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "SCAN.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include <iostream>
+
+namespace rajaperf
+{
+namespace algorithm
+{
+
+
+void SCAN::runOpenMPVariant(VariantID vid)
+{
+#if defined(RAJA_ENABLE_OPENMP) && defined(RUN_OPENMP)
+
+  const Index_type run_reps = getRunReps();
+  const Index_type ibegin = 0;
+  const Index_type iend = getRunSize();
+
+  SCAN_DATA_SETUP;
+
+  switch ( vid ) {
+
+#if _OPENMP >= 201811
+    case Base_OpenMP : {
+
+      startTimer();
+      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+        SCAN_PROLOGUE;
+        #pragma omp parallel for reduction(inscan, +:scan_var)
+        for (Index_type i = ibegin; i < iend; ++i ) {
+          y[i] = scan_var;
+          #pragma omp scan exclusive(scan_var)
+          scan_var += x[i];
+        }
```

Comment:
@trws I'm having trouble with the openmp variants where I'm using openmp scan. Clang11 on blueos does not appear to give the right answers for the SCAN kernel here, and it segfaults in the INDEXLIST kernel.

Reply:
@trws On a similar not is OpenMP scan supposed to work with target team distribute?

Reply:
I have worked around the openmp scan issues I was seeing by disabling the code with a configure flag. We can revisit this when scan is better supported.

Reply:
@MrBurmark please make an issue so we don't forget to circle back to this.